apps/drec-ui/src/device/logic/smartMeter/table/smartMeterTable.ts

```diff
@@ -0,0 +1,23 @@
+import { EnergyFormatter } from '@energyweb/origin-ui-utils';
+import dayjs from 'dayjs';
+import { BigNumber } from 'ethers';
```

Comment:
We should use `@ethersproject/bignumber` instead of full 'ethers' since we are using only BigNumber part of it.